inotify_poller.go

```diff
@@ -38,7 +38,6 @@ func newFdPoller(fd int) (*fdPoller, error) {
 			poller.close()
 		}
 	}()
-	poller.fd = fd
```

Comment:
Just to be super explicit about this, line 35 calls `emptyPoller(fd)` and on line 24 `poller.fd = fd`